tests/main/interfaces-rtc/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: Check that RTC device nodes are accessible through an interface
+
+details: |
+    This test makes sure that a snap using the RTC interface can access
+    the /dev/rtc device node exposed by a slot on the OS snap properly.
+
+systems: [ubuntu-core-16-64]
+
+prepare: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+
+    echo "Given a snap declaring a plug on rtc is installed"
+    . $TESTSLIB/snaps.sh
+    install_local rtc-consumer
+
+    echo "And the rtc plug is connected"
+    . $TESTSLIB/names.sh
+    snap connect rtc-consumer:rtc ${core_name}:rtc
```

Comment:
You can connect with just `snap connect rct-consumer:rtc :rtc` (once https://github.com/snapcore/snapd/pull/2520 lands)